doc/api/esm.md

```diff
@@ -437,10 +437,34 @@ The `--experimental-json-modules` flag is needed for the module
 to work.
 
 ```bash
-node --experimental-modules --entry-type=module index.js # fails
-node --experimental-modules --entry-type=module --experimental-json-modules index.js # works
+node --experimental-modules index.mjs # fails
```

Comment:
This change seems unrelated. On the other hand, it would be helpful to include an example of using WASM as an entry type.

Reply:
Thanks, let's merge https://github.com/nodejs/node/pull/27678 first then.